### ISSUE TITLE
How to calculate accuracy during training?

### ISSUE THREAD

**Issue by sachinkmohan** [pierluigiferrari/ssd_keras#366]
How to calculate accuracy during training?

I tried the below in the code in ssd7_training.ipynb file and didn't work. Any help would be awesome!

`model.compile(optimizer=adam, loss=ssd_loss.compute_loss, metrics=['accuracy'])`

        * Operating System - Ubuntu 18.04
        * Keras version - 2.2.5
        * TensorFlow version - TF 1.14

**Comment by vk1996** [comment]
Object detection is not evaluated with accuracy , instead you can use meanAveragePrecision . You can check out my project to see how mAP can be evaluated after every epoch using keras callbacks

**Comment by Lucky2593** [comment]
This is a regression problem, I think that's why we only can get a loss for the problem. 

NOTE: You can add the following line :- 
model.compile(optimizer=sgd,loss=ssd_loss.compute_loss, metrics=[tf.keras.metrics.Accuracy()])
But you will get accuracy as "00+e00".(because it's a regression problem.)
